flank-scripts/build.gradle.kts

```diff
@@ -119,7 +119,9 @@ val prepareJar by tasks.registering(Copy::class) {
 
 val download by tasks.registering(Exec::class) {
     commandLine(
-        "gh", "release", "download", "flank-scripts-$version", "-D", System.getenv("GITHUB_WORKSPACE") ?: ".."
+        // todo for test purposes only!!!! (workaround for temporarily unavailable newer version)
```

Comment:
please use it locally with https://github.com/cli/cli instead of pushing hacks 😃

Reply:
That was a temporary solution (for veryfication) and will not be part of PR